prio/client.c

```diff
@@ -336,6 +336,64 @@ PrioClient_encode(const_PrioConfig cfg, const bool* data_in,
   return rv;
 }
 
+// Encode long as bools and write them to an array with offset.
+SECStatus
+long_to_bool(bool* dst, long x, int prec, int entry)
```

Comment:
Could you add some unit tests for this function? This is probably the most subtle part of the PR, so it'd be nice to get a bit more confidence that it's bug-free.

One hacky (but functional) way to test non-public functions is to just `#include "prio/client.c" in the test.

Reply:
Will do.

Reply:
Tests are done, tell me if you think they are sufficient.

Reply:
One could also change this to use unsigned long, so there would be one more compiler check. On the other hand, we will feed it output from fixed point and additively shifted negative number functions, so we would likely need to do type conversions or casting, so doing range checks P_CHECKCB on longs seems preferable. Also it is using long now, and does the correct thing, so we would not need to change it. What do you think?